server/test/src/scala/lsug/ServerSpec.scala

```diff
@@ -0,0 +1,86 @@
+package scala.lsug
+
+import cats.effect.{Blocker, IO}
+import lsug.protocol.Meetup.MeetupDotCom
+import lsug.protocol.Meetup.MeetupDotCom.Event
+import lsug.{IOSuite, Meetup, Server, protocol}
+
+import java.nio.file.Paths
+import java.time.{LocalDate, LocalDateTime, LocalTime, ZoneId, ZonedDateTime}
+
+final class ServerSpec extends IOSuite {
+
+  class MockedMeetup extends Meetup[IO] {
```

Comment:
Strictly speaking, this isn't a mock object as there's no behaviour verification. There's a good [summary of the differences between mocks, stubs and fakes by Martin Fowler](https://martinfowler.com/articles/mocksArentStubs.html).

Could you rename this to `FakeMeetup`?